### ISSUE TITLE
TypeError: bip32.fromSeed is not a function

### ISSUE THREAD

**Issue by random9brat** [bitcoinjs/bip32#62]
Trying to get bech32 address from mnemonic words and im getting this error:

const root = bip32.fromSeed(seed)
^

TypeError: bip32.fromSeed is not a function

this is the code im running: 

```
var  randomBytes = crypto.randomBytes(16)
var mnemonic = bip39.entropyToMnemonic(randomBytes.toString('hex')) 
const seed = bip39.mnemonicToSeedSync(mnemonic); 
const root = bip32.fromSeed(seed); 
```
  thanks for any helps since im new in the bitcoinjs and crypto programming in general

**Comment by junderw** [comment]
Closed as duplicate: #61 

Please read the README and previous issues before making an issue.

**Comment by random9brat** [comment]
I did but it seems i miss understoud somehting, still cant get bech32 easily from 12 words... did try bitcoinjs/bip32 node js example and also bitcoinjs/bech32 but it seems im missing something...

also i could get to work with this top bip32.fromSeed(seed) code working either...

Could use some help :/
